templates/hoover-deps.nomad

```diff
@@ -524,7 +524,70 @@ job "hoover-deps" {
   }
   {% endif %}
 
-  group "search-rabbitmq" {
+  {% if config.snoop_nlp_entity_extraction_enabled or config.snoop_nlp_language_detection_enabled %}
+   group "nlp-service" {
+    count = ${config.snoop_nlp_count}
+
+    ${ continuous_reschedule() }
+    ${ group_disk() }
+
+    task "nlp" {
+      ${ task_logs() }
+
+      constraint {
+        attribute = "{% raw %}${meta.liquid_volumes}{% endraw %}"
+        operator = "is_set"
+      }
+
+      driver = "docker"
+      config {
+        image = "${config.image('nlp-service')}"
+        volumes = [
+          "{% raw %}${meta.liquid_volumes}{% endraw %}/nlp-service/data:/data",
+        ]
+        port_map {
+          nlp = 5000
+        }
+        labels {
+          liquid_task = "hoover-nlp"
+        }
+        memory_hard_limit = ${4 * config.snoop_nlp_memory_limit}
+      }
+      env {
+        NLP_SERVICE_PRESET = "${config.snoop_nlp_preset}"
+        NLP_SERVICE_FALLBACK_LANGUAGE = "${config.snoop_nlp_fallback_language}"
+        NLP_SPACY_TEXT_LIMIT = "${config.snoop_nlp_spacy_text_limit}"
+        GUNICORN_WORKERS = ${config.snoop_nlp_gunicorn_workers}
+        GUNICORN_THREADS = ${config.snoop_nlp_gunicorn_threads}
+        EXTRACT_ENTITIES = "${config.snoop_nlp_entity_extraction_enabled}"
+        DETECT_LANGUAGE = "${config.snoop_nlp_language_detection_enabled}"
+      }
+      resources {
+        memory = ${config.snoop_nlp_memory_limit}
+        cpu = 1500
+        network {
+          mbits = 1
+          port "nlp" {}
+        }
+      }
+      service {
+        name = "hoover-nlp-service"
+        tags = ["fabio-/_nlp strip=/_nlp"]
+        port = "nlp"
+        check {
+          name = "http"
+          initial_status = "critical"
+          type = "http"
+          path = "/config"
+          interval = "${check_interval}"
+          timeout = "600s"
```

Comment:
Why do we need such a huge timeout? Is the `/config` route slow? 

Let's put the usual value `${check_timeout}` here, and if it fails, use a faster route for `/config`, or set up more workers/threads for gunicorn

Reply:
It was this huge, because the nlp-service needs so long to set up and download the models. The `/config` route is only available afterwards.